requirements.txt

```diff
@@ -4,3 +4,4 @@ plotly == 4.1.0
 pynumpress >= 0.0.4
 regex
 ms_deisotope == 0.0.9
+dash
```

Comment:
Please specify the exact version.
I know that I did not do that for e.g. regex, but we should really start doing that and maybe get the renovate bot (https://github.com/renovatebot) to update our dependencies to stay stable :)

Reply:
I agree